centos-7.3-x86_64.json

```diff
@@ -1,7 +1,7 @@
 {
   "builders": [{
     "type": "qemu",
-    "iso_url": "{{user `mirror`}}/7/isos/x86_64/CentOS-7-x86_64-NetInstall-1611.iso",
+    "iso_url": "{{user `mirror`}}/7/isos/x86_64/CentOS-7-x86_64-NetInstall-1708.iso",
```

Comment:
1708.iso is for CentOS 7.4, not 7.3.
the old iso is at `/7.3.1611/isos/x86_64/CentOS-7-x86_64-NetInstall-1611.iso`.
`meta/main.yml` in ansible-role-* treats all centos-7.x as `EL 7` and i think we should move from centos-7.x to centos-7 to avoid bumping versions in ansible-role-*.

Reply:
would you elaborate? preferably in an issue.